common/OpTestHost.py

```diff
@@ -1219,3 +1219,35 @@ def host_list_usb_devices(self):
     #
     def host_enable_single_core(self):
         self.host_run_command("ppc64_cpu --cores-on=1")
+
+    ##
+    # @brief This function disables kdump service
+    #
+    # @param os_level: string output of /etc/os-release
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def host_disable_kdump_service(self, os_level):
+        if "Ubuntu" in os_level:
+            self.host_run_command("service kdump-tools stop")
+            self.host_run_command("service kdump-tools status")
+        else:
+            self.host_run_command("service kdump stop")
+            self.host_run_command("service kdump status")
```

Comment:
Where ever appropriate use 'systemctl' command. Command 'service' is deprecated.